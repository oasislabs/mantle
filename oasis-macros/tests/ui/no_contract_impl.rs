oasis_std::contract! {

#[derive(Contract)]
pub struct Counter(u32);

}

fn main() {}
