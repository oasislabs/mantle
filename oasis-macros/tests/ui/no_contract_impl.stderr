error: Missing implementation for `Counter::new`.
 --> $DIR/no_contract_impl.rs:4:12
  |
4 | pub struct Counter(u32);
  |            ^^^^^^^

error: aborting due to previous error

